Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "polyline"
 description = "Encoder and decoder for the Google Encoded Polyline format"
-version = "0.11.0"
```

Comment:
personal preference/nit: leave out the version bump, but please do add a BREAKING change log release.  

It's too hard to remember if versions have already been bumped by someone else in another PR since release, so we just bump upon release, depending on what's in the changelog.